server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -2130,10 +2140,6 @@ private void snapshotFile(BlobStoreIndexShardSnapshot.FileInfo fileInfo, IndexId
                 final long partBytes = fileInfo.partBytes(i);
 
                 InputStream inputStream = new InputStreamIndexInput(indexInput, partBytes);
-                if (snapshotRateLimiter != null) {
```

Comment:
This was already removed back in https://github.com/elastic/elasticsearch/commit/e65c60915a2ce870162af65747a652706f8b2043 - looks like we missed it.